theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/HorizontallyScrollableArea.tsx

```diff
@@ -201,6 +203,33 @@ function useHandlePanAndZoom(
           )
         })
 
+        // Set maximum scroll points based on the sequence length.
+        // This is to avoid zooming out to infinity.
+        const sequenceLength = val(layoutP.sheet).getSequence().length
```

Comment:
Wow it's really cool to see dataverse code in a PR 🤩

Reply:
Don't know what that means, but I'm just winging it based on the existing code base 😁

Is dataverse the ’val()’ stuff?

Reply:
Yeah